### DISCUSSION TITLE
notes/frontend/js/bosn/chapter01

### DISCUSSION THREAD

**Discussion by giscus** [FuckDoctors/notes2#4] [Announcements]
# notes/frontend/js/bosn/chapter01

第 1 章 数据类型 1-1 深入浅出课程介绍 略 1-2 六种数据类型 原始类型（基本类型）： number string boolean null undefined 除此之外，JavaScript 还有一种对象类型（复合类型）object。

https://zhaobc.gitee.io/notes/frontend/js/bosn/chapter01.html

**Comment by kfpvdt** [comment]
想把blog加个在线编辑器搜到的大佬，看了下大佬写的那个组件。大佬牛批

**Reply by FuckDoctors** [reply]
过奖啦，也是新手，主题作者 Mr.Hope 才是真牛
